external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/PartitionManager.java

```diff
@@ -74,8 +74,8 @@ public void ack(String offset) {
   @Override
   public void fail(String offset) {
     logger.warn("fail on " + offset);
```

Comment:
this might trigger lot of log writing if the spout starts failing. May be good idea to make it debug